### DISCUSSION TITLE
Is there a mkdocs equivalent for sphinx autosummary?

### DISCUSSION THREAD

**Discussion by sotte** [mkdocstrings/mkdocstrings#484] [Ideas]
Sphinx has this nice feature to generate [summary tables of classes/functions/etc](https://www.sphinx-doc.org/en/master/usage/extensions/autosummary.html). This is nice if you e.g. have a module that has many classes/functions and you want to get a quick overview of what is available.

Now the question: does `mkdocstrings` have an equivalent to this?

![image](https://user-images.githubusercontent.com/79138/195035723-4f6463e3-c002-423b-807a-1c729b979dbc.png)

**Comment by pawamoy** [comment]
Hi, no, there's no equivalent to `autosummary` (yet?) :confused:

**Comment by pawamoy** [comment]
mkdocstrings-python now has a `summary` option, see https://mkdocstrings.github.io/python/usage/configuration/members/#summary :slightly_smiling_face:
